src/mon/MonCommands.h

```diff
@@ -242,20 +242,6 @@ COMMAND("mon ok-to-rm " \
 	"check whether removing the specified mon would break quorum",
 	"mon", "r")
 
-COMMAND_WITH_FLAG("mon_status", "report status of monitors", "mon", "r",
-	     FLAG(NOFORWARD))
-COMMAND_WITH_FLAG("sync force " \
-	"name=yes_i_really_mean_it,type=CephBool,req=false " \
-	"name=i_know_what_i_am_doing,type=CephBool,req=false", \
-	"force sync of and clear monitor store", \
-        "mon", "rw", \
-        FLAG(NOFORWARD)|FLAG(DEPRECATED))
-COMMAND_WITH_FLAG("heap " \
```

Comment:
```
======================================================================
FAIL: test_ceph_argparse.TestMonitor.test_heap
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/pybind/test_ceph_argparse.py", line 340, in test_heap
    self.assert_valid_command(['heap', 'dump'])
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/pybind/test_ceph_argparse.py", line 53, in assert_valid_command
    assert_not_in(result, [{}, None])
AssertionError: {} unexpectedly found in [{}, None]

```

"heap" command is removed from the registry. so we need to remove the corresponding tests from `test_ceph_argparse.py`.

Reply:
There is a way to find these at make check, downside would be the need to add the required frameworks and generated APIs.